src/framework/preferences/css/stylus/utils/Themes.styl

```diff
@@ -4,23 +4,38 @@
 themes = {
     ".fl-theme-bw": {
         foregroundColor: #000000,
-        backgroundColor: #ffffff
+        backgroundColor: #ffffff,
+        .fl-prefsEditor-buttons { //FLUID-4991: The shadow effect around the hide/show preferences bug fix.
```

Comment:
I'm getting an error when trying to build the stylus files. I believe that the class name might need to be wrapped in `"`